configs/karaf/alpaca.script

```diff
@@ -8,4 +8,6 @@ feature:install islandora-connector-broadcast
 feature:install islandora-indexing-triplestore
 feature:install islandora-indexing-fcrepo
 
+feature:install islandora-http-client
+
```

Comment:
Can you add a `feature:install fcrepo-api-x` in this script as well?

Also, can you bump the version number to 0.2.0 for the api-x feature?